lib/mongo_storage.js

```diff
@@ -0,0 +1,49 @@
+var db = require('monk');
+
+module.exports = function(path) {
```

Comment:
Thanks @lambtron! I'm reviewing and testing this now and we should have it merged soon. Thanks for your patience.